renderdoc/driver/vulkan/wrappers/vk_cmd_funcs.cpp

```diff
@@ -7570,6 +7570,36 @@ void WrappedVulkan::vkCmdEndRendering(VkCommandBuffer commandBuffer)
   }
 }
 
+VkResult WrappedVulkan::vkCreateDeferredOperationKHR(VkDevice device,
+                                                     const VkAllocationCallbacks *pAllocator,
+                                                     VkDeferredOperationKHR *pDeferredOperation)
+{
+  return ObjDisp(device)->CreateDeferredOperationKHR(Unwrap(device), pAllocator, pDeferredOperation);
```

Comment:
It would be good to have a brief comment here to explain why we're not intercepting and wrapping these objects, just to clarify for the future.

Reply:
Done.